application/features/Term.py

```
@@ -77,37 +82,45 @@
         self.term = None
 
     def handleMessage(self):
```

Ruff

Missing return type annotation for public function `handleMessage`

Add return type annotation: `None`

(ANN201)